MIGRATION.md

```diff
@@ -302,6 +304,14 @@
   - [Packages renaming](#packages-renaming)
   - [Deprecated embedded addons](#deprecated-embedded-addons)
 
+## From version 7.3.0 to 7.4.0
+
+#### CommonJS with Vite is deprecated
+
+Using CommonJS in the `main` configuration with `main.cjs` or `main.cts` is deprecated, and will be removed in Storybook 8.0. This is a necessary change because Vite will remove support for CommonJS in the upcoming v5 release.
```

Comment:
@benmccann is there a reference we can link to here, that describes this breaking change in Vite v5?

Reply:
At the moment the only link is: https://github.com/vitejs/vite/discussions/13928

I would just say "an upcoming release" because while I'm aiming for Vite 5 I'm not 100% sure whether it will be Vite 5 or 6.